docs/README.md

```diff
@@ -1,68 +1,89 @@
+</p>
+
+<p align="center">
+    <em>Prompt Engineering focused on developer experience</em>
+</p>
+
+<p align="center">
+    <a href="https://github.com/Mirascope/mirascope/actions/workflows/tests.yml" target="_blank"><img src="https://github.com/Mirascope/mirascope/actions/workflows/tests.yml/badge.svg?branch=main" alt="Tests"/></a>
+    <a href="https://docs.mirascope.io/" target="_blank"><img src="https://img.shields.io/badge/docs-available-brightgreen" alt="Docs"/></a>
+    <a href="https://pypi.python.org/pypi/mirascope" target="_blank"><img src="https://img.shields.io/pypi/v/mirascope.svg" alt="PyPI Version"/></a>
+    <a href="https://github.com/Mirascope/mirascope/stargazers" target="_blank"><img src="https://img.shields.io/github/stars/Mirascope/mirascope.svg" alt="Stars"/></a>
+</p>
 
 ---
 
-[![GitHub stars](https://img.shields.io/github/stars/Mirascope/mirascope.svg)](https://github.com/Mirascope/mirascope/stargazers)
-[![Documentation](https://img.shields.io/badge/docs-available-brightgreen)](https://docs.mirascope.io/)
-[![](https://github.com/Mirascope/mirascope/actions/workflows/tests.yml/badge.svg?branch=main)](https://github.com/Mirascope/mirascope/actions/workflows/tests.yml)
-[![GitHub issues](https://img.shields.io/github/issues/Mirascope/mirascope.svg)](https://github.com/Mirascope/mirascope/issues)
-[![Github discussions](https://img.shields.io/github/discussions/Mirascope/mirascope)](https:github.com/Mirascope/mirascope/discussions)
-[![GitHub license](https://img.shields.io/github/license/Mirascope/mirascope.svg)](https://github.com/Mirascope/mirascope/blob/main/LICENSE)
-[![PyPI version](https://img.shields.io/pypi/v/mirascope.svg)](https://pypi.python.org/pypi/mirascope)
-[![PyPI pyversions](https://img.shields.io/pypi/pyversions/mirascope.svg)](https://pypi.python.org/pypi/mirascope)
+**Documentation**: <a href="https://docs.mirascope.io" target="_blank">https://docs.mirascope.io</a>
+
+**Source Code**: <a href="https://github.com/Mirascope/mirascope" target="_blank">https://github.com/Mirascope/mirascope</a>
 
 ---
 
 ## Why use Mirascope?
 
-- **Easy**: Designed for ease of use, Mirascope features an intuitive interface and a minimal learning curve
-- **Intuitive**: Get editor support for prompts, eliminating the need to dig through documentation
-- **Durable**: Mirascope offers versatile and adaptable functionality, allowing seamless integration with custom solutions
-- **Integration**: Leveraging Pydantic, Mirascope offers easy integration with JSON Schema and other tools
-- **Maintainability**: Reduce prompt-related bugs and organize prompts with provided utilities
+**Mirascope** is a library purpose-built for Prompt Engineering on top of <a href="https://pydantic.dev" target="_blank">Pydantic</a>:
 
-## Installation
+* **Intuitive**: Editor support that you expect (e.g. **autocompletion**, **inline errors**)
```

Comment:
I like this line a lot because it tells you more directly than the previous iteration what are the problems mirascope is solving. That being said, it doesn't feel as in your face for the following lines - maybe I'm too inexperienced in the LLM field to take my word on this (or alternatively, I should be the gold standard for when something is obvious enough) but the following lines don't mean much to me since I haven't been building LLMs long, let alone with other libraries. Maybe they could be even more in your face?

Reply:
Like none of the following lines really make 100% to me for the following reasons: 
Peace of Mind: what kind of prompt-related bugs and what feature of Pydantic? My assumption is validation lets us not get silly things passed into chat models but could be clearer. 
Durable: extend what functionality? And why would I need to maintain a fork otherwise?
Integration makes sense
Convenience: why would I need to maintain other tools
Open makes sense

maybe this is just showing my greenness with LLMs and its not indicative of the average user but just sharing my two cents

Reply:
@brenkao I was writing this geared more towards senior developers. Does the language here scratch the itch, or should it be more clear? Any suggestions for things that might cover what @brianpark0126 mentioned?